test/helpers/screenshot_test.exs

```diff
@@ -4,10 +4,73 @@ defmodule ScreenshotTest do
 
   hound_session()
 
-  test "should take a screenshot" do
+  setup do
     navigate_to("http://localhost:9090/page1.html")
-    path = take_screenshot("screenshot-test.png")
+    :ok
+  end
+
+  test "should take a screenshot with a generated file name" do
+    default_path = File.cwd!()
+
+    path = take_screenshot()
+
     assert File.exists?(path)
+    assert path =~ default_path
+    assert file_name(path) =~ ~r/^screenshot-\d{4}(?:-\d{1,2}){5}\.png$/
   end
 
+  test "file names generated 1 second apart are unique" do
+    default_path = File.cwd!()
+
+    path1 = take_screenshot()
+    :timer.sleep(1000)
```

Comment:
This adds an extra second to an already slow test, so I'm happy to rip this test out if you don't think it's necessary. I couldn't think of a better way to test this.

Reply:
Tests speed are not that much of an issue yet, but is the granularity of the file name 1 second?

Reply:
@tuvistavie yeah, based on the way the filename is currently generated, it will only be unique down to the second.

https://github.com/marcdel/hound/blob/9a627eab709e5b852ee08eb2934e1fc37296128b/lib/hound/helpers/screenshot.ex#L33

Reply:
Right, I completely forgot we were generating the name this way. Thanks.
We should probably improve this but it's not related to this PR, so let's see later.